yomi/README.md

```diff
@@ -8,3 +8,18 @@ TODO(kashomon): describe how this works.
 
 * [Kubernetes+Redis+WQ](https://kubernetes.io/docs/tasks/job/fine-parallel-processing-work-queue/)
 * [Redis+Guestbook Example](https://github.com/kubernetes/examples/tree/master/guestbook)
+
+# Some notes:
+
+Resources for using generated client libraries. Note that documentation says
```

Comment:
"generated client libraries" for which clients of what services?  A subject pronoun here is nice.